plugins/woocommerce/includes/admin/meta-boxes/views/html-order-item.php

```diff
@@ -47,7 +49,16 @@
 
 	<?php do_action( 'woocommerce_admin_order_item_values', $product, $item, absint( $item_id ) ); ?>
 
-	<td class="item_cost" width="1%" data-sort-value="<?php echo esc_attr( $order->get_item_subtotal( $item, false, true ) ); ?>">
+	<?php if ( wc_get_container()->get( CostOfGoodsSoldController::class )->feature_is_enabled() ) : ?>
+		<td class="item_cost" width="1%" data-sort-value="<?php echo esc_attr( $item->get_cogs_value() ); ?>">
```

Comment:
I'm not too sure we should hijack the `item_cost` CSS class as there might be 3rd party scripts or even just styles targeting what now would be the "Price" column as it has always used the other class. I know it's a bit confusing, but I'd suggest we use a new CSS class for the COGS column and despite the new "Price" name leave `item_cost` as the CSS class for the usual price column.

What do you think?

Reply:
Done, I've reverted the existing class to the original `item_cost` name, and renamed the new class to `item_cost_of_goods`.